src/components/userLocations.vue

```diff
@@ -164,22 +172,15 @@ export default defineComponent({
         UPDATE(location: UserLocation){
             this.UPDATE_USER_LOCATION(location)
             .catch(()=>{
-                var index = _.findIndex(this.user_locations, {id: location.id});
-                var index_copy = _.findIndex(this.user_locations_copy, {id: location.id});
-                Object.assign(this.user_locations_copy[index_copy], this.user_locations[index]) 
+                this.user_locations_copy = JSON.parse(JSON.stringify(this.user_locations))
```

Comment:
As we are already using Lodash, you could use it for cloning:

`this.user_locations_copy = _.cloneDeep(this.user_locations);`
 
It has certain advantages over JSON.parse&stringify:

- https://lodash.com/docs/#cloneDeep
- https://www.samanthaming.com/tidbits/70-3-ways-to-clone-objects/